### ISSUE TITLE
textDescription option should be removed

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/tasks#908]
While reviewing https://github.com/phetsims/sun/issues/332 I noticed that textDescription option is used in VerticalCheckBoxGroup, and a quick project search shows it appearing a number of places. This a11y option is quite old and isn't used anymore, all usages should be deleted.

**Comment by jessegreenberg** [comment]
I am seeing usages in forces-and-motion-basics, joist, and sun.

**Comment by jessegreenberg** [comment]
Removed, closing.
